src/Service.GraphQLBuilder/Mutations/CreateMutationBuilder.cs

```diff
@@ -126,11 +126,29 @@ private static InputValueDefinitionNode GenerateSimpleInputType(NameNode name, F
                 defaultValue = value.Fields[0].Value;
             }
 
+            ITypeNode type;
+            if (defaultValue is not null)
+            {
+                type = f.Type.NullableType();
+            }
+            else
+            {
+                // ID field cannot be nullable.
+                if (f.Type.NamedType().Name.Value != "ID")
+                {
+                    type = f.Type.NullableType();
+                }
```

Comment:
This is new logic, not clear if/how it relates to the function support changes. This also needs a test to validate